configure

```diff
@@ -83,7 +83,7 @@ CONFIGURED=""
 if [ "$DEST_DIR_SET" -ne 0 ]
 then
   case $DEST_DIR in 
-    (/*) pathchk -- "$1";; 
+    (/*) pathchk "$DEST_DIR" || exit -1 ;;
```

Comment:
Great catch!

I would suggest we keep the `--` to demarcate the end of command line options to `pathchk` to handle the (weird) case of the path starting with a `-`.

I also think it would be nice of us to echo an error message on top of the one we get from `pathchk` to help the user understand what's going wrong.

```suggestion
    (/*) pathchk -- "$DEST_DIR" || { echo "error: invalid directory name for --chpl-home: $DEST_DIR"; exit -1; };; 
```

Reply:
@ShreyasKhandekar the case of a path starting with a `-` is already prevented by the case statement, this is the case where the path begins with `/`. Thus the `--` is unnecessary and misleading.

Reply:
Oh I see, you're right!

Reply:
Explicit error message added as suggested in  1d6704e8